handler/delete.go

```diff
@@ -41,10 +42,12 @@ func Delete(ms storage.MetricStore) func(http.ResponseWriter, *http.Request, htt
 			labels, err := splitLabels(labelsString)
 			if err != nil {
 				http.Error(w, err.Error(), http.StatusBadRequest)
+				log.Debug(err.Error(), http.StatusBadRequest)
```

Comment:
This could be a bit hard to understand among other logs, I'd suggest formatting these with "Failed to parse URL %v: %v" etc.